src/content/blog/2024_qcoder.md

```diff
@@ -0,0 +1,163 @@
+---
+title: QCoder - A platform for quantum competitive programming
+author: Kein Yukiyoshi
+day: 20
+month: 10
+year: 2024
+tags: 
+  - quantum algorithms
+  - qiskit
+  - python
+---
+
+<p align="center">
+    <img src='../../images/2024-qcoder/qcoder_logo.png' alt='QCoder Logo' width=800/>
+</p>
+
+In recent years, quantum algorithms have attracted attention for their theoretical promise of significant efficiency gains.
+Shor's algorithm, for example, can factor integers with exponentially fewer queries than known classical algorithms.
+While learning about these algorithms is exciting, it can be challenging to stay motivated—especially for those without an academic background who are learning just for fun.
+Textbooks and academic papers are often too complex for beginners, making it hard to follow along.
+Even if you succeed in understanding and implementing an algorithm, running it on today’s quantum computers in scalable settings is typically impossible.
+
+In response to these challenges, we have developed <a href="https://www.qcoder.jp/" target="_blank">**QCoder**</a>, a platform for quantum competitive programming.
+QCoder is the quantum counterpart to conventional competitive programming platforms such as Codeforces, Google Code Jam, and ICPC.
+It allows users to learn quantum algorithms and practice quantum programming simply by participating in competitions.
+
+# Getting Started
+
+## Step 1: Environment Setup
+we currently support <a href="https://github.com/Qiskit/qiskit/" target="_blank">Qiskit</a> as the available quantum programming language.
```

Comment:
*We

Reply:
Fixed: https://github.com/unitaryfund/unitary.fund/pull/553/commits/17a978f2d6f18b92bac8ee2a96c082502c02e87b